src/ray/common/constants.h

```diff
@@ -13,8 +13,9 @@ constexpr int kObjectIdIndexSize = 32;
 static_assert(kObjectIdIndexSize % CHAR_BIT == 0,
               "ObjectID prefix not a multiple of bytes");
 
+constexpr int64_t kActorIdSize = 4;
```

Comment:
Java currently uses the return object ID of the actor creation task as actor ID, that would need to be changed as well? Similar changes are needed for `TaskInterface` in C++ core worker which does translations between ObjectID and ActorID.